.travis.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       php: "7.1"
       before_install: []
       before_script:
-        - pecl install -f mongodb-${DRIVER_VERSION}
```

Comment:
I actually don't recall why we stopped using the `tpecl` command, which caches the install. It looks like it was removed in https://github.com/mongodb/mongo-php-library/commit/8774590ef047b9997cd245ece7dfcd83edc63e16.

`.travis.yml` still has the `tpecl` definition, but it looks like it's no longer being used since that commit. Do you want to try relocating it to `install-extension.sh` and use it when installing based on `$DRIVER_VERSION`? Otherwise, I suppose we should just delete it entirely (can do so in a separate issue/PR).

Reply:
I'll change it back to using `tpecl`, makes sense to cache it when possible.